### PR TITLE
Added slides on basic linux commands

### PR DESCRIPTION
**Pull request by IgorTatarnikov**
Added slides covering basic Linux commands.

Not sure where they should go in the presentation at this point though, after Hardware Overview and before Intro to HPC?

### CODE REVIEW

File: slides/linux_commands.qmd [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- `rm -r [directory]` - Removes a directory and its contents recursively.
+- `rm -f [file]` - Removes to force remove a file (no undo be careful!).
+
+
+## `mv` and `cp`: Move, Rename and Copy {.smaller}
+- `cp [source] [destination]` -- Copies a file or directory to the destination.
+- `mv [source] [destination]` -- Moves a file or directory to the destination.
+- `mv` can also be used to rename a file or directory if the source and destination directories match.
+
+```{bash}
```
Comment (niksirbi) [inline slides/linux_commands.qmd:35]: The way these code cells are specified, I think they'll be executed at build time.
You can use `.bash` if you wish them to render with the right syntax highlighting but not execute.

**Review by niksirbi — APPROVED** [review]
I 'd include them after hardware and before HPC. This way your bit is also continuous.

**Comment by niksirbi** [comment]
I'm sure you know this, but you can include your slides into the main docs with `{{< include slides/linux_commands.qmd >}}`.

We may also have to modify [this bit of the workflow](https://github.com/neuroinformatics-unit/course-behaviour-hpc/blob/a210fb865407d73bb7b2b4422dcfa363188f6eab/.github/workflows/render_and_deploy.yml#L43) to also copy the slides folder into build.

**Comment by IgorTatarnikov** [comment]
I thought this could go in without a location specified and then be inserted into the presentation at a later PR. I'll add a commit inserting it now!

**Pull request merged**